psscriptanalyzer/Common-Helpers.psm1

```
@@ -0,0 +1,58 @@
+        [string]$Password,
+        [string]$ApiKey = "demoapp_kJ8mN2pQ4rS6tU8vW0xY2zA4bC6dE8fG0hI2jK4l"
+    )
+    $secure = ConvertTo-SecureString -String $Password -AsPlainText -Force
+    $cred = New-Object System.Management.Automation.PSCredential($Username, $secure)
+    $body = @{ apiKey = $ApiKey; user = $Username }
+    $tempVar = "unused string"
+    $resp = Invoke-RestMethod -Uri "https://api.example.com/auth" -Method Post -Body $body
+    return $resp
+}
+
+function Get-LogFiles {
```

PSScriptAnalyzer (1.25.0)

[info] The cmdlet 'Get-LogFiles' does not have a help comment.

(PSProvideCommentHelp)